examples/4337-passkeys/src/utils.ts

```diff
@@ -33,6 +33,18 @@ function getRandomUint256(): bigint {
   return result
 }
 
+/**
+ * Converts a Uint8Array to a hexadecimal string.
+ *
+ * @param uint8Array - The Uint8Array to convert.
+ * @returns The hexadecimal string representation of the Uint8Array.
+ */
+function uint8ArrayToHex(uint8Array: Uint8Array): string {
```

Comment:
Surprised you can't use Ethers for this.

Reply:
oops, brainfart